### PR TITLE
Adding docs for install and using-inventory

### PR DESCRIPTION
**Pull request by sacharya**

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+```
+minikube start -p hive
+```
+
+Deploy hive to the hub cluster following [hive docs](https://github.com/openshift/hive/blob/master/docs/developing.md).
+```
+git clone https://github.com/openshift/hive
+cd hive
+git checkout master
+make build
```
Comment (rwsu) [inline docs/install.md]: Before "make build" I also had to "sudo dnf install golang-github-mock", otherwise it couldn't find the 'mockgen' command.

Comment (sacharya) [reply docs/install.md]: Added

File: docs/install.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+make deploy
+
```
[the remark this answers is not in the document]
Comment (sacharya) [reply docs/install.md]: Done.

File: docs/install.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+```
+Adopt cluster0 as a managed cluster
+```
+kubectl create namespace cluster0
+bin/hiveutil create-cluster --adopt --adopt-admin-kubeconfig=/tmp/cluster0.config --adopt-cluster-id=cluster0 --adopt-infra-id=cluster0 -n cluster0 cluster0
+```
+Note: You may need to add --include-secrets=false as documented [here](https://github.com/openshift/hive/issues/774) if you get an error message about missing AWS credentials.
+
+/tmp/cluster0.config is the kubeconfig file for the managed cluster "cluster0" with embedded certs.
```
[the remark this answers is not in the document]
Comment (sacharya) [reply docs/install.md]: Done

**Pull request merged**